src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsIpc/IpcTransport.cs

```diff
@@ -48,6 +48,88 @@ internal abstract class IpcEndpoint
         public abstract Task WaitForConnectionAsync(CancellationToken token);
     }
 
+    internal class IpcEndpointConfig
```

Comment:
This should be moved to its own file. It has no relationship to IpcEndpoint and its derived classes.

Reply:
Didn't think it was that off since its the configuration used together with IpcEndpoint's, but could of course be move to a different file.